src/Common/FieldVisitorConvertToNumber.h

```diff
@@ -53,7 +52,6 @@ class FieldVisitorConvertToNumber : public StaticVisitor<T>
 
     T operator() (const UInt64 & x) const { return T(x); }
     T operator() (const Int64 & x) const { return T(x); }
-    T operator() (const Int128 & x) const { return T(x); }
```

Comment:
Here were leftovers from when we used UInt128 for UUID a long time ago. I've cleaned it up. Some bugs can be automatically fixed by this change...